incubator/monochart/templates/gateway.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.gateway -}}
```

Comment:
@yciabaud  thanks for the PR!
Maybe better to call the file `istio_dateway.yaml` since `istio` is not the only gateway possible.
Have you tested it?

Thanks